### PR TITLE
feat(integrations): add support for Dixa

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Add support for Dixa

## Issue ticket number and link
[EXT-163](https://linear.app/nango/issue/EXT-163/add-support-for-dixa)

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -1523,6 +1523,29 @@ discourse:
...
 
+dixa:
+    display_name: Dixa
+    categories:
+        - support
+    auth_mode: API_KEY
+    proxy:
+        base_url: https://${connectionConfig.apiType}.dixa.io
+        headers:
+            Authorization: ${apiKey}
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:1725]: Can be tested maybe

Comment (hassan254-prog) [reply packages/shared/providers.yaml:1725]: Should we enforce verification on untested integrations on our end for both [Dixa](https://github.com/NangoHQ/nango/pull/2899) and [Chattermill](https://github.com/NangoHQ/nango/pull/2900). Could this be a case of shooting ourselves in the foot 🤔?

Comment (bodinsamuel) [reply packages/shared/providers.yaml:1725]: I mean when you publish an integration I have no way to know if it's tested or not, if you don't feel confident (that's okay) you should write it in the PR desc ☺️

**Review by bodinsamuel — APPROVED** [review]

File: packages/shared/providers.yaml (hunk 1 above)
Comment (hassan254-prog) [reply packages/shared/providers.yaml:1725]: I am sorry for the miscommunication here🤦‍♂️. Thank you for the clarification 🙏.

**Pull request merged**